packages/feature-toggle/tsconfig.typecheck.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
So we are not typechecking other projects for tests?

Reply:
there are type errors in tests in other project now so I'm not going to enable it now